internal/layer2/announcer.go

```diff
@@ -196,13 +196,13 @@ func (a *Announce) gratuitous(ip net.IP) error {
 	if ip.To4() != nil {
 		for _, client := range a.arps {
 			if err := client.Gratuitous(ip); err != nil {
-				return err
+				level.Error(a.logger).Log("gratuitousAnnounce", "error", err, "ip", ip, "msg", "failed to make gratuitous IP announcement")
```

Comment:
Would it make sense to return here?

Reply:
I meant return nil

Reply:
but then it would skip sending the GARP on all other interfaces?

Reply:
we should probably pick up discussion on #974